### ISSUE TITLE
Move override scripts to respective repos

### ISSUE THREAD

**Issue by dblock** [opensearch-project/opensearch-build#153]
**Is your feature request related to a problem? Please describe.**

Custom bundle build scripts in https://github.com/opensearch-project/opensearch-build/tree/main/bundle-workflow/scripts/components need to be moved into individual repositories.

**Describe the solution you'd like**

Add these scripts to each repo [under `scripts/build.sh`](https://github.com/opensearch-project/opensearch-build/blob/main/tools/bundle-build/lib/component.py#L39), and delete them from https://github.com/opensearch-project/opensearch-build/tree/main/scripts/bundle-build/components.

**Additional context**

Leftover from #150, part of https://github.com/opensearch-project/opensearch-build/issues/133

**Comment by gaiksaya** [comment]
[Triage] @dblock is this isssue still valid? Looks like Meta issue is closed

**Comment by dblock** [comment]
Yes, we still want this, i.e. deleting everything under https://github.com/opensearch-project/opensearch-build/tree/main/scripts/components.

**Comment by gaiksaya** [comment]
Hi @prudhvigodithi ,

This isse is different than the one you specified. This is about moving all the component scripts, build.sh and integtest.sh scripts to respective repositories than maintaining in build repo which might different in version. See db's comment above.

**Comment by prudhvigodithi** [comment]
In past I have migrated `build.sh` to respective repos, https://github.com/opensearch-project/opensearch-build/pull/2471#issuecomment-1218224133. We should list of what is remaining start to create a campaign.

**Comment by gaiksaya** [comment]
I still below component scripts in build repo: https://github.com/opensearch-project/opensearch-build/tree/main/scripts/components
* OpenSearch
* functionalTestDashboards 
* notifications-core
* OpenSearch-Dashboards    
* alerting                 
* notifications            
* performance-analyzer

@peterzhuamazon Wondering if we can move below components scripts to their own repo:
* OpenSearch
* OpenSearch-Dashboards    
* alerting                         
* performance-analyzer
